atlp-rwanda/falcons-ec-be
# #184637277 Password Renewal

**### What does this PR do?**
This pull request adds a feature that prompts users to update their passwords when they expire.

**### Description of Task to be completed?**
As a user , I should be prompted to update my password
Such that the information about the last time password reset be recorded


**### How should this be manually tested?**
To test this feature manually, the server should be run and a cron job will execute periodically to check if any passwords have expired. If there are any expired passwords, the user IDs and statuses will be displayed on the terminal, and the user statuses will be changed to "NeedToUpdatePassword". To confirm that the feature is working correctly, check the content of the Users table in the database.

**### Any background context you want to provide?**
None
**### What are the relevant pivotal tracker/Trello stories?**
Tracker: 184637277

### **Screenshots (if appropriate)**
none

**### Questions:**
none
